src/test/kotlin/org/rust/cargo/project/model/impl/SyncToolWindowTest.kt

```diff
@@ -316,6 +318,79 @@ class SyncToolWindowTest : RsWithToolchainTestBase() {
         """)
     }
 
+    @MinRustcVersion("1.62.0")
+    @WithExperimentalFeatures(EVALUATE_BUILD_SCRIPTS)
+    fun `test Cargo continues build script evaluation after a compilation error in an independent crate`() {
+        if (SystemInfo.isWindows) return // It looks like error messages are a bit different on Windows
```

Comment:
It would be nice to check it on Windows as well some days. Could you add some `TODO` here, please?